src/actions/breakpoints/addBreakpoint.js

```diff
@@ -16,10 +17,11 @@ export default async function addBreakpoint(
   const state = getState();
 
   const source = getSource(state, breakpoint.location.sourceId);
+  const _source = source.toJS();
```

Comment:
i think `sourceRecord` and `source` would be clearer, but it's a new precident so we can also just not use `_source` and do `source.toJS()` everywhere too...